QUICKSTART/QUICKSTART.md

```diff
@@ -0,0 +1,258 @@
+
+# Quickstart for VSCODE-CODE-COVERAGE
+
+**Summary**:  This is the QUICKSTART.md document to help users get started with the `ryanluker.vscode-coverage-gutters` "*Coverage Gutters*" extension for VS Code.
+
+## Terminology
+
+*Coverage Gutters* depends uses two files.  It's easy to get the files mixed up and we need to make sure that everone is on the same page when we talk about these files.
+
+<details>
+<summary>Coverage File</summary>
+
+-----
+
+  * LCOV or XML file containing detailed coverage data
+
+* *Coverage Gutters* supports LCOV, CLOVER, COBERTURA, and JACOCO coverage files
+
+* Generation of the coverage file is language specific and is beyond the scope of this extension.
```

Comment:
Maybe mention after this that they could look at the examples for scenarios on how to generate the coverage files.
https://github.com/ryanluker/vscode-coverage-gutters/tree/master/example/ruby#ruby